### PR TITLE
Update yarn and dependencies

### PR DESCRIPTION
**Pull request by dovicsin**

### CODE REVIEW

**Review by veghdev — DISMISSED** [review]

File: plugins/data-types/src/utils/mainTypeConverter.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { clearValue } from './clearValue'
 
 export const convertToNumber = (values: string[] | number[] | undefined): number[] => {
 	if (!values) return []
-	return values.map((value) => {
+	return values.map((value: string | number | undefined) => {
```
Comment (brsd) [inline plugins/data-types/src/utils/mainTypeConverter.ts:5]:
```suggestion
	return values.map((value: string | number) => {
```

File: plugins/data-types/src/utils/mainTypeConverter.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const convertToNumber = (values: string[] | number[] | undefined): number
 
 export const convertToString = (values: string[] | number[] | undefined): string[] => {
 	if (!values) return []
-	return values.map((value) => {
+	return values.map((value: string | number | undefined) => {
```
Comment (brsd) [inline plugins/data-types/src/utils/mainTypeConverter.ts:16]:
```suggestion
	return values.map((value: string | number) => {
```

File: plugins/csv-parser/src/headerDetect.ts [hunk 3]
```diff
@@ -22,20 +22,20 @@ const getType = (value: string | number): string => {
 			.replace(/[\u2012\u2013\u2014\u2015]/g, '-')
 
 		return !isNaN(Number(formattedValue)) ? 'number' : 'string'
```
Comment (brsd) [inline plugins/csv-parser/src/headerDetect.ts:24]: Maybe the whole getType fn is duplicated?
https://github.com/vizzuhq/vizzu-lib-ext/pull/58/files#diff-630ed1549ccd46bc82d93e2a9323013ba377fa3fabc45e473d616b9747615779R18

File: package.json [hunk 4]
```diff
@@ -1,63 +1,40 @@
 {
 	"private": true,
-	"packageManager": "yarn@4.0.2",
+	"packageManager": "yarn@4.3.1",
```
Comment (brsd) [inline package.json:3]: Why to use `yarn@4.3.1` instead of the latest `yarn@4.5.0`?

File: plugins/csv-parser/src/headerDetect.ts (hunk 3 above)
Comment (dovicsin) [reply plugins/csv-parser/src/headerDetect.ts:24]: because they are two separate plugins completely independent from each other.

**Review by veghdev — DISMISSED** [review]

**Review by brsd — APPROVED** [review]

**Pull request merged**